jsonnet/kube-prometheus/jsonnetfile.json

```diff
@@ -72,7 +72,7 @@
           "subdir": "docs/node-mixin"
         }
       },
-      "version": "master"
+      "version": "v0.18.1"
```

Comment:
Is there a reason why we are using a tag instead of the branch `release-0.18` here?

Reply:
actually noticed that this does not work at all, but somehow our CI did not catch this. Before with make generate-in-docker we ran jb update/install now we skip this step so when running make generate it did not catch this, I am already fixing this.

Reply:
I looked into it and it appears that we never ran `jb update` in the CI: https://travis-ci.org/github/coreos/kube-prometheus/jobs/674766820#L261.
It is not linked to moving away from `generate-in-docker` but to the lack of `jb update`

Reply:
yes, true! My mistake, this should fix it https://github.com/coreos/kube-prometheus/pull/502